6_handbook/20_application_tuning.txt

```diff
@@ -208,3 +208,16 @@ mount /var/rudder/cfengine-community/state
 
 ----
 
+==== Rsyslog
+
+On Rudder policy servers (root or relay), when managing a large number of nodes,
+you can experience issues with rsyslog. This happens because Rudder uses TCP
```

Comment:
It is only configurable since 3.1, I will make a PR to adapt this section to 3.1 (and add a note in the upgrade section about it).